integrations/amazon_bedrock/src/haystack_integrations/components/generators/amazon_bedrock/adapters.py

```diff
@@ -140,7 +138,6 @@ def prepare_body(self, prompt: str, **inference_kwargs) -> Dict[str, Any]:
             body = {"messages": [{"role": "user", "content": prompt}], **params}
         else:
             default_params = {
-                "max_tokens_to_sample": self.max_length,
```

Comment:
I guess to be consistent with how we are handling the other default params is to set this to the appropriate default value that's used in Bedrock. What do you think?